python/ray/data/_internal/batcher.py

```diff
@@ -49,11 +49,12 @@ class Batcher(BatcherInterface):
     # zero-copy views, we sacrifice what should be a small performance hit for better
     # readability.
 
-    def __init__(self, batch_size: Optional[int]):
+    def __init__(self, batch_size: Optional[int], zero_copy: bool = True):
```

Comment:
Can you document what does it means exactly "zero copy" for this class?

Reply:
I'll add a docstring for the constructor!